css/css-values/support/mixed-units-07.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      body {
+        margin: 0;
+        font-size: 16px;
+      }
+      .box {
+        background: rgb(0, 0, 255);
+        width: 100%;
```

Comment:
You can remove `width: 100%` and just let it stretch.

In fact you could even add `class="box"` to the `<body>`, rely on background propagation, and then remove `margin: 0` and `height: 10px`:

```html
<!DOCTYPE html>
<style>
  .box {
    background: rgb(0, 0, 255);
  }
  @media (width: calc(100px / 1em * 1em)) {
    .box {
      background: rgb(255, 165, 0);
    }
  }
</style>
<body class="box"></body>
```

But feel free to keep `<div class="box"></div>` if you don't want to rely on background propagation.

Reply:
Same response from https://github.com/web-platform-tests/wpt/pull/42517#discussion_r1362927521

> @Loirooriol I was matching the values set in previous neighboring tests. Should I remove those declarations for all of them?
> * [mixed-units-01.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-01.html)
> * [mixed-units-02.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-02.html)
> * [mixed-units-03.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-03.html)
> * [mixed-units-04.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-04.html)
> * [mixed-units-05.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-05.html)
> * [mixed-units-06.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-06.html)

Reply:
Resolved in [9c784ec](https://github.com/web-platform-tests/wpt/pull/42517/commits/9c784ec62bf53ecd2f38483e145342142cf1e7e8)